src/components/MoviesList/MoviesList.tsx

```diff
@@ -3,20 +3,20 @@ import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
 interface Props {
-  movies: Movie[];
+  movies: Movie[] | null;
```

Comment:
it couldn't be null
<img width="386" alt="image" src="https://user-images.githubusercontent.com/64437769/160362376-cdac0f1b-be39-453b-aad5-65c4aa30d5b8.png">